src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -160,26 +242,56 @@ class hitTest {
         const items = this.findItems(clickLocation[0], clickLocation[1]);
 
         const overToggleTrack = items.filter(x => x.actionType === "toggleTrack").length > 0;
+
+        const currentPreviewEtlItem = items.filter(x => x.actionType === "previewEtlScript").map(x => x.arg as previewEtlScriptItemContext)[0];
```

Comment:
let's refactor all items to use find instead of filter.

Reply:
done